software/youtubedl-material.yml

```diff
@@ -9,6 +9,7 @@ platforms:
 tags:
   - Automation
 source_code_url: https://github.com/Tzahi12345/YoutubeDL-Material
+depends_3rdparty: true
```

Comment:
```suggestion
```

"a Material Design frontend for [youtube-dl](https://rg3.github.io/youtube-dl/)." The README also mentions downloading from other supported sites